RangePicker/RangePicker.js

```diff
@@ -120,7 +141,7 @@ const RangePickerBase = kind({
 	render: ({value, ...rest}) => {
 		return (
 			<PickerCore {...rest} index={0} value={value}>
```

Comment:
I think we are missing `type="number"` here.

Reply:
Hm.. Think we don't have `type` prop for `internal/Picker`.

Reply:
I will add it and make calculations based on it